### ISSUE TITLE
Unit chosen to find actual frequency

### ISSUE THREAD

**Issue by pratibha77118** [ubsuny/23-Homework5G3#42]
@s4il3sh 
It seems like you have chosen units as cycle per decades. Can you clearify more on it?

**Comment by s4il3sh** [comment]
> @s4il3sh It seems like you have chosen units as cycle per decades. Can you clearify more on it?

Yes, you are right! But my code is not complete. After fft they frequency seems higher so, i guess units will drop to cyles per month or something like that. How did you guys do it?

**Comment by uarif** [comment]
BTW, I chose per 6 months and I guess that makes more sense, as there is almost 20 or more  ups and downs in 10 years, so

**Comment by reshnashrestha** [comment]
Yeah, I can agree with @uarif

**Comment by pratibha77118** [comment]
I do agree with @uarif . If you choose decades then there won't be much data .

**Comment by uarif** [comment]
Yes, That is what I think

**Comment by LinxuanHu** [comment]
agree

**Comment by reshnashrestha** [comment]
we chose cycles per year.
